src/rollup/globals/reset-default-props.js

```diff
@@ -0,0 +1,128 @@
+/** @module pathfora/globals/reset-default-props */
+
```

Comment:
lol i'd never really noticed this but might be worth making a ticket about handling defaults more gracefully. this approach seems like it might smell a little and be super difficult to scale as new features are added. nothing to do here just a note.